pkg/cfn/manager/nodegroup.go

```diff
@@ -5,14 +5,13 @@ import (
 	"fmt"
 
 	"github.com/aws/aws-sdk-go/aws"
-	"github.com/pkg/errors"
-	"github.com/weaveworks/eksctl/pkg/cfn/builder"
-	"github.com/weaveworks/eksctl/pkg/eks/api"
-
 	cfn "github.com/aws/aws-sdk-go/service/cloudformation"
-	"github.com/kubicorn/kubicorn/pkg/logger"
+	"github.com/kris-nova/logger"
```

Comment:
originally we tried to keep imports in logical groups, but clearly those have lost meaning over time and got messy... :(

Reply:
Sure, this is being done automatically by `goimports` and my IDE.